newrelic/structures_newrelic_nrql_alert_condition.go

```diff
@@ -72,6 +74,13 @@ func expandNrqlAlertConditionCreateInput(d *schema.ResourceData) (*alerts.NrqlCo
 		}
 	}
 
+	if conditionType == "baseline" {
+		if attr, ok := d.GetOk("signal_seasonality"); ok {
+			seasonality := alerts.NrqlSignalSeasonality(strings.ToUpper(attr.(string)))
+			input.SignalSeasonality = &seasonality
+		}
+	}
+
```

Comment:
```suggestion
	if conditionType == "baseline" {
		if attr, ok := d.GetOk("signal_seasonality"); ok {
			seasonality := alerts.NrqlSignalSeasonality(strings.ToUpper(attr.(string)))
			input.SignalSeasonality = &seasonality
		} else {
			seasonality := alerts.NrqlSignalSeasonalities.NewRelicCalculation
			input.SignalSeasonality = &seasonality
		}
	}
```

I'll explain why I suggest this modification in the following comment (just under this).